### PR TITLE
Sheets get cohort

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Some small-ish changes to the documentation for this method (it seemed fairly complete to me already).

### CODE REVIEW

File: api/sheets/get_cohort_members.md [hunk 1]
```diff
@@ -6,22 +6,24 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the values of a given property for a node.
+# Retrieve the values of a given property for a node
```
Comment (beets) [inline api/sheets/get_cohort_members.md:9]: Same comment on the page title

File: api/sheets/get_cohort_members.md [hunk 2]
```diff
@@ -6,22 +6,24 @@ parent: Google Sheets
...
 
-## `=DCCOHORTMEMBERS(dcids)`
+Get the [members](http://browser.datacommons.org/kg?dcid=member) of each [cohort](/glossary.html) provided. Here a cohort is a general term for a group of entities, like [the CDC's list of the United States' 500 largest cities](https://datacommons.org/browser/CDC500_City).
 
-Given a [cohort](http://browser.datacommons.org/kg?dcid=Cohort) or a row of cohort DCIDs, get the [members](http://browser.datacommons.org/kg?dcid=member) of each cohort.
+## General information about this method
+
+**Signature:** `=DCCOHORTMEMBERS(dcids)`
```
Comment (beets) [inline api/sheets/get_cohort_members.md:15]:
```suggestion
**Formula:** `=DCCOHORTMEMBERS(dcids)`
```

**Review by beets — CHANGES_REQUESTED** [review]
looks good! minor comments.

File: api/sheets/get_cohort_members.md (hunk 1 above)
Comment (KilimAnnejaro) [reply api/sheets/get_cohort_members.md:9]: Discussion: https://github.com/datacommonsorg/docsite/pull/127#discussion_r571358818

**Review by beets — APPROVED** [review]
thx!

**Pull request merged**